common/source/docs/common-arkflow.rst

```diff
@@ -0,0 +1,53 @@
+.. _common-arkflow:
+
+================================
+ARK Flow Open Source Optical Flow and Distance Sensor
+================================
+
+[copywiki destination="copter,plane"]
+
+The `ARK Flow Open Source Optical Flow and Distance Sensor <https://arkelectron.com/product/ark-flow/>`__ is an open source optical flow sensor including a Broadcom AFBR lidar which uses the CAN protocol to communicate with the autopilot.  This can be used to improve horizontal position control especially in GPS denied environments.
+
+.. image:: ../../../images/arkflow/ark_flow.jpg
+   :target: ../_images/arkflow/ark_flow.jpg
+   :width: 450px
+
+..  youtube:: SAbRe1fi7bU
+    :width: 100%
+
+Where to Buy
+------------
+
+The sensor is available from `ARK Electronics <https://arkelectron.com/product/ark-flow/>`__
+
+Connection to Autopilot
+-----------------------
+
+- The ARK Flow is connected to the CAN bus using a Pixhawk standard 4 pin JST GH cable.
+- Multiple sensors can be connected by plugging additional sensors into the ARK Flow's second CAN connector.
+- The recommended mounting orientation is with the connectors on the board pointing towards back of vehicle, as shown in the following picture.
+
+.. image:: ../../../images/arkflow/ark_flow_orientation.png
+   :target: ../_images/arkflow/ark_flow_orientation.png
```

Comment:
Are we safe to use the figure on the right side of the image? Copyright wise? Most likely yes. Just checking.

Reply:
We can check with @hamishwillee as I originally pulled this from the PX4 docs.

Reply:
The strict rules for copying content are that you attribute according to the source license - for PX4 using CC. However there is some wriggle room - for example, if this image originates ultimately from arkflow marketing material (which is quite possible) and was shared by ark, then arkflow can provide permission to reshare it without attribution under some other license.

This is why I encourage vendors to do the first post of their materials under the site license.

I am _fairly_ sure you could find the same image under arkflow docs.